esti/webhook_server_test.go

```diff
@@ -103,6 +103,7 @@ func hookHandlerFunc(respCh chan hookResponse) func(http.ResponseWriter, *http.R
 			return
 		}
 
+		time.Sleep(1 * time.Second) // Added sleep to differentiate between event timestamps
```

Comment:
I hate it and understand why we need it.
The funny thing is: the test may still fail due to the same reason..